src/test/java/petStoreTests/user/DeleteUserNegativeTest.java

```diff
@@ -0,0 +1,30 @@
+package petStoreTests.user;
+
+import client.UserClient;
+import data.DataSet;
+import io.qameta.allure.Description;
+import io.restassured.response.Response;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import petStoreTests.AbstractTest;
+
+import java.io.IOException;
+
+public class DeleteUserNegativeTest extends AbstractTest {
+
+    @Description("Verify that we can't delete non existing user from the store data base")
+    @Test
+    public void deleteUserTest() throws IOException {
+
+        LOGGER.info("BEFORE TEST making sure that the base can't delete non existing user");
+        UserClient.postUser(DataSet.addingUser());
+        UserClient.deleteUserByUsername(DataSet.addingUser().getUsername());
+        LOGGER.info("BEFORE TEST USER ADDED AND DELETED");
+
+        LOGGER.info("START TEST delete non existing user from the store data base");
+        Response response = UserClient.deleteNonExistingUser(DataSet.addingUser().getUsername());
+
+        Assert.assertTrue(response.asString().trim().isEmpty());
```

Comment:
I don't really understand this assertion

Reply:
@andyyarych when there is no user to delete i'm getting status code 404 and empty response. So here i'm waiting for it)